src/pages/Location/LocationConfirmation/LocationConfirmation.js

```diff
@@ -76,7 +99,6 @@ const LocationConfirmation = () => {
                   Disclaimer: Cover crop recommendations are based on expert opinions. Your cover
                   crop performance and seeding rates will vary based on location, management,
                   cultivars, and many other variables. Consult your local
-                  {' '}
```

Comment:
same deal here, it now says `Consult your local[NRCS Service Center](https://www.nrcs.usda.gov/wps/portal/nrcs/detailfull/national/programs/financial/csp/?&cid=nrcsdev11_000242),[Cooperative Extension Service office](https://nifa.usda.gov/land-grant-colleges-and-universities-partner-website-directory), or[Conservation District](https://www.nacdnet.org/general-resources/conservation-district-directory/)for detailed guidance.`

Reply:
notice the missing spaces